clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -48,7 +48,8 @@ export async function activate(context: vscode.ExtensionContext) {
 
     // Listeners
     context.subscriptions.push(vscode.workspace.onDidChangeConfiguration(event => {
-        if (event.affectsConfiguration(SETTINGS_SECTION)) {
+        if (event.affectsConfiguration(SETTINGS_SECTION) &&
+        vscode.workspace.getConfiguration().get(SETTINGS_SECTION + ".profiles")) {
```

Comment:
Couple of questions: 
- can we use ```vscode.workspace.getConfiguration(SETTINGS_SECTION).get("profiles")```?
- is mandatory have the ```.``` before profile?

Reply:
good idea, I will modify it